### ISSUE TITLE
With the valorant elo 

### ISSUE THREAD

**Issue by ItsFloBTW** [RumbleMike/ValorantStreamOverlay#91]
When i put the settings to my user and password and refresh it does not change

**Comment by tylerana** [comment]
Same here! I lost a game, but my RP & ELO are stuck at 99. It does not show my recent games either.

**Comment by HolidayLNX** [comment]
> Same here! I lost a game, but my RP & ELO are stuck at 99. It does not show my recent games either.

Is your rank Iron 1? The accounts with Iron 1 has this problem.. not sure why.. My Iron 2 account had the same problem .. but when promoted to iron 2 .. its working

**Comment by tylerana** [comment]
Yes it was! My friend boosted it to iron 2. It took him 4 games and he had to get approx. 30 frags each game. It sucks he had to do it, but I'm glad it's working now.

**Comment by mohnishkarri** [comment]
guys this issue is closed already. Refer to #17
